packages/server/modules/workspaces/graph/resolvers/workspaceJoinRequests.ts

```diff
@@ -45,5 +56,37 @@ export default {
     workspace: async (parent, _args, ctx) => {
       return await ctx.loaders.workspaces!.getWorkspace.load(parent.workspaceId)
     }
+  },
+  Mutation: {
+    workspaceJoinRequestMutations: () => ({})
+  },
+  WorkspaceJoinRequestMutations: {
+    approve: async (_parent, args) => {
```

Comment:
you could use the new `commandFactory` here, it handles the transaction management for you